unzip/README.md

```diff
@@ -0,0 +1,57 @@
+# unzip command
+
+This is a simple `unzip` command.
+It's meant to be used with the binary-compatibility mode of Unikraft.
+
+The contents are:
+
+* /usr/bin/unzip: pre-built Linux ELF
+* lib/, lib64/: pre-extracted dynamic libraries required to run the `unzip`
+
+
+## (Re)Extract Dynamic Libraries
+
+You can see the list of dynamics libraries for the `unzip` command using the ldd command:
+```
+ldd $(which unzip)
+ linux-vdso.so.1 (0x00007ffed119f000)
+ libbz2.so.1.0 => /lib/x86_64-linux-gnu/libbz2.so.1.0 (0x00007f6154470000)
+ libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f6154248000)
+ /lib64/ld-linux-x86-64.so.2 (0x00007f61545ab000)
+```
+
+ 
+You can copy the dynamic libraries required to run the `unzip` by using:
+
+```console
+cp --parents $(which unzip) /lib/x86_64-linux-gnu/libbz2.so.1.0 /lib/x86_64-linux-gnu/libc.so.6 /lib64/ld-linux-x86-64.so.2 .
+```
+
+The command output signals the copying of the required dynamic libraries:
```

Comment:
This makes sense only if using [`extract.sh`](https://github.com/unikraft/dynamic-apps/blob/master/extract.sh). When you use the `cp` command above, it doesn't make sense to add this.